numpy/lib/tests/test_io.py

```diff
@@ -637,6 +637,64 @@ def check_large_zip(memoryerror_raised):
             pytest.xfail("subprocess got a SIGKILL, apparently free memory was not sufficient")
         assert p.exitcode == 0
 
+    def test_fstring_fmt(self):
```

Comment:
There are only tests with float format (e.g. `{:.3f}`). Could you add binary format as well? (this resolves #20755)